content/docs/pipelines/sdk/client_usage.md

```diff
@@ -0,0 +1,43 @@
++++
+title = "Use Kubeflow Pipelines API Client"
```

Comment:
Our sdk includes kfp.compiler, kfp.component, kfp.dsl and kfp.client (listed at https://www.kubeflow.org/docs/pipelines/sdk/sdk-overview/) . This page only helps user to leverage the sdk client (aka api client). "Using the Kubeflow Pipelines SDK" seems a little bit too generic, e.g., the other pages on building components are also using sdk. WDYT?

Reply:
The docs refer to this client as being part of the KFP SDK. The KFP API refers to the REST API in the docs. Since this example uses the client from the SDK, it should refer to the SDK even though it is using the API behind the scenes.

Reply:
done